ci-pack.ps1

```diff
@@ -3,4 +3,4 @@ dotnet clean -c Release
 $repositoryUrl = "https://github.com/$env:GITHUB_REPOSITORY"
 
 # Building for packing and publishing.
-dotnet pack -c Release --output "$PSScriptRoot/artifacts" /p:RepositoryUrl=$repositoryUrl
+dotnet pack -c Release -p:PackageOutputPath="$PSScriptRoot/artifacts" -p:RepositoryUrl=$repositoryUrl
```

Comment:
Not sure why we used `/p` before. The docs say `-p` 

https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-pack#examples